src/code/bank1.asm

```diff
@@ -891,21 +891,15 @@ MinimapEntryPoint::
 
 jr_001_5639::
     JP_TABLE
-    ; Code below is actually data for the jump table
-    ld   c, b
-    ld   d, [hl]
-    ld   a, b
-    ld   d, [hl]
-    db   $F4 ; Undefined instruction
-    ld   d, [hl]
-    db   $FD ; Undefined instruction
-    ld   d, [hl]
-    dec  bc
-    ld   d, a
-    inc  e
-    ld   d, a
-    dec  h
-    ld   e, b
+._00 dw func_001_5648
+._01 dw jr_001_5678
+._02 dw jr_001_56F4
+._03 dw jr_001_56FD
+._04 dw jr_001_570B
```

Comment:
Thanks for finding this missing jump table!

Usually, I tend to format jump table handlers not much as `jr_` labels, but rather either as functions (`func_001_570B`) or named handlers (like `MinimapState04Handler`, at least before it can get understood and renamed to something more meaningful).

But if you want to merge it like this, it's also fine with me 👍